catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/ChartRepository.java

```diff
@@ -126,9 +128,11 @@ private EntityReference getService(Chart chart) throws IOException, ParseExcepti
 
   public static class ChartEntityInterface implements EntityInterface<Chart> {
     private final Chart entity;
+    private final String fqn;
```

Comment:
why do we want to keep this around as member?

Reply:
Fixed it.

Reply:
@vivekratnavel not sure what was the fix. But my question was around why we need to have "fqn" as the member variable why not make that call and get it on demand?

Reply:
Yes, that's what I am doing with the latest patch. Getting the fqn on demand - https://github.com/open-metadata/OpenMetadata/pull/2608/files#diff-a981d13766f798ba878ac3c8867295329e36539829b1b5f6cad3b9fa28b81aa3R163